src/api/organization.js

```diff
@@ -15,8 +18,182 @@ export const deleted = function (query) {
   });
 };
 
+export const pending = function (query) {
+  return axios_cancelable.get(`${config.dataApi}/organization/pending?${qs.stringify(query)}`, {
+    headers: setHeaders()
+  });
+};
+
 export const nonPublishing = function (query) {
   return axios_cancelable.get(`${config.dataApi}/organization/nonPublishing?${qs.stringify(query)}`, {
     headers: setHeaders()
   });
+};
+
+export const getOrganization = key => {
+  return axios.get(`${config.dataApi}/organization/${key}`, {
+    headers: setHeaders()
+  });
+};
+
+export const getHostedDatasets = ({ key, query }) => {
+  return axios.get(`${config.dataApi}/organization/${key}/hostedDataset?${qs.stringify(query)}`, {
+    headers: setHeaders()
+  });
+};
+
+export const getPublishedDatasets = ({ key, query }) => {
+  return axios.get(`${config.dataApi}/organization/${key}/publishedDataset?${qs.stringify(query)}`, {
+    headers: setHeaders()
+  });
+};
+
+export const getInstallations = ({ key, query }) => {
+  return axios.get(`${config.dataApi}/organization/${key}/installation?${qs.stringify(query)}`, {
+    headers: setHeaders()
+  });
+};
+
+export const createOrganization = data => {
+  return axios.post(`${config.dataApi}/organization`, data, {
+    headers: setHeaders()
+  });
+};
+
+export const updateOrganization = data => {
+  return axios.put(`${config.dataApi}/organization/${data.key}`, data, {
+    headers: setHeaders()
+  });
+};
+
+export const getOrganizationOverview = async key => {
+  const organization = (await getOrganization(key)).data;
+  const publishedDataset = (await getPublishedDatasets({ key, query: {} })).data;
```

Comment:
As far as I can see these are only used for counts. Setting a limit=0 on the query would reduce number of downloaded kb significantly.

Reply:
Sorry, I haven't thought about such an approach. Thank you for the piece of advice.